crates/pipeline-manager/src/compiler.rs

```diff
@@ -640,6 +674,8 @@ inherits = "release"
                                 }
                             },
                             Err(DBError::UnknownPipeline { .. }) => {
+                                info!("Pipeline no longer exists, canceling compilation job: program {} version {} (tenant: {})",
```

Comment:
```suggestion
                                debug!("Pipeline no longer exists, canceling compilation job: program {} version {} (tenant: {})",
```

Reply:
Made `debug` in all three places where this message occurs.